src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/StandardDirectionOrderDJ.java

```diff
@@ -911,39 +909,6 @@ private DynamicList deleteLocationList(DynamicList list) {
         return DynamicList.builder().value(list.getValue()).build();
     }
 
-    private LocationRefData fillPreferredLocationData(final List<LocationRefData> locations,
-                                                      DynamicList caseDataList) {
-        if (Objects.isNull(caseDataList) || Objects.isNull(locations)) {
-            return null;
-        }
-        String locationLabel = caseDataList.getValue().getLabel();
-        var preferredLocation =
-            locations
-                .stream()
-                .filter(locationRefData -> checkLocation(
-                    locationRefData,
-                    locationLabel
-                )).findFirst();
-        return preferredLocation.orElse(null);
-    }
-
-    private Boolean checkLocation(final LocationRefData location, String locationTempLabel) {
-        String locationLabel = location.getSiteName()
-            + " - " + location.getCourtAddress()
-            + " - " + location.getPostcode();
-        return locationLabel.equals(locationTempLabel);
-    }
-
-    private DynamicList getLocationListFromCaseData(DynamicList hearingList, DynamicList trialList) {
```

Comment:
are these functions being moved somewhere or were they removed as they were unused?

Reply:
yes fillPreferredLocationData and getLocationListFromCaseData are unused methods